src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  &__image {
+    height: 40px;
+    align-self: center;
```

Comment:
Same here. You first give the container the align-items and then you repeat the same property in the flex item